CommunityShedMVC/src/CommunityShedMVC/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@model CommunityShedMVC.ViewModels.LoginViewModel
+
+@{
+    ViewBag.Title = "Login";
+}
+
+<h2>Login</h2>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add a `ValidateAntiForgeryToken` attribute to the POST action method.